trailblazer/store/models.py

```diff
@@ -84,7 +83,7 @@ class Analysis(Model):
     @property
     def has_ongoing_status(self):
         """Check if the log has an ongoing status: 'running|'pending'"""
```

Comment:
```suggestion
        """Check if the log has an ongoing status: 'running|'pending|completing'"""
```
right?

Reply:
and Error. I will remove the options though and leave it as `Check if the log has an ongoing status.`